_posts/2023-09-25-concurrent_segment_search.markdown

```diff
@@ -0,0 +1,140 @@
+---
+layout: post
+title:  "Introducing concurrent segment search in OpenSearch"
+authors: 
+  - reta
+  - sohami
+date:   2023-09-25
```

Comment:
Please update the date to: 2023-10-18

Reply:
@sohami - If you can make these small updates now, I can have the team push your blog live.

Reply:
done